pyccel/stdlib/numpy/numpy_f90.f90

```diff
@@ -0,0 +1,86 @@
+! --------------------------------------------------------------------------------------- !
+! This file is part of Pyccel which is released under MIT License. See the LICENSE file   !
+! or go to https://github.com/pyccel/pyccel/blob/master/LICENSE for full license details. !
+! --------------------------------------------------------------------------------------- !
+
+module numpy_f90
+
+    use ISO_C_BINDING
+    
+    implicit none
+    
+    interface numpy_sign
```

Comment:
This new keyword should also be added to the fortran name clash checker

Reply:
https://github.com/pyccel/pyccel/pull/1169/commits/0fcbccaba0791ab93f4070a2cca89a76263b973f

Reply:
https://github.com/pyccel/pyccel/pull/1169/commits/0fcbccaba0791ab93f4070a2cca89a76263b973f https://github.com/pyccel/pyccel/pull/1169/commits/253338b4f6440c2e33a9566c7d459d7040bb8be8